CHANGELOG.md

```diff
@@ -18,7 +18,11 @@ See this project's [releases](/../../../releases).
 <!-- ### Security -->
 <!-- ### Notes -->
 
-## [Unreleased]
```

Comment:
Hiya sorry, this title is of equal level to the version titles below and is empty as I haven't locally developed any changes that are yet to be released. But if for example, I added a new feature X and pushed to github, I'd note it here and then when the new version of the plugin is released, this would automatically get set as the new version title. i.e. Since your changes aren't released as a new version yet, I would put them under 'Unreleased' and when a new version is pushed they would automatically be listed under the latest version. Does that make sense? (Not sure this is the best method since this is the only repo i have in any level of 'production' status but it's worked for me so far so if that's not traditionally what's done, always happy to update).

Reply:
Ohh gotcha that makes sense! I'd missed the section above where it's mentioned that the changelog is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).

I can think of a couple slightly less ambiguous approaches to this (and e.g. [Common Changelog](https://common-changelog.org/#62-how-is-this-different-from-keep-a-changelog) avoids the Unreleased section entirely for a couple reasons), but since this is already following Keep a Changelog, it's probably better to keep it as-is than to diverge from that documented format.

Reverted back!